pkg/minikube/bootstrapper/bsutil/versions_test.go

```diff
@@ -70,7 +70,7 @@ func TestVersionIsBetween(t *testing.T) {
 			ver:         semver.MustParse("1.8.0-alpha.0"),
 			gte:         semver.MustParse("1.8.0"),
 			lte:         semver.MustParse("1.9.0"),
-			expected:    true,
+			expected:    false,
```

Comment:
why this was changed to false?

Reply:
Should the alpha version be between those two versions? I'm not too sure what the actual expectation is here, but the reason I changed the expected values was because the assertion fails otherwise. In both cases I believe changing the expectation does make sense, but let me know if I'm missing something.

Reply:
Hi @renatolabs, you are correct in that the expected value should be `false`. Would you mind updating the test description to `alpha before` as that's actually what the test case is.

Also, for the following test `beta greater than alpha`, it's impossible for any version to pass as the `gte` and `lte` are backwards. Would you mind swapping those and changing the expected to `true`.

One that change is made I'd be happy to merge this PR, thanks

Reply:
Updated!